File: app/src/main/java/com/neome/api/meta/base/dto/SchemaColumnMap.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.EnumDefnCompType;
import com.neome.api.meta.base.Types.MetaIdField;
import com.neome.api.meta.base.Types.SymbolColumn;

import java.util.Map;

@SuppressWarnings("unused")
public class SchemaColumnMap
{
  public Map<SymbolColumn, SchemaColumn> columnMap;

  public MetaIdField fieldId;

  public EnumDefnCompType fieldType;
}

File: app/src/main/java/com/neome/api/meta/base/dto/SchemaColumnMap.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.EnumDefnCompType
import java.util.Map
import com.neome.api.meta.base.Types.MetaIdField
import com.neome.api.meta.base.dto.SchemaColumn
import com.neome.api.meta.base.Types.SymbolColumn

open class SchemaColumnMap
{
  lateinit var columnMap: Map<SymbolColumn, SchemaColumn>
  lateinit var fieldId: MetaIdField
  lateinit var fieldType: EnumDefnCompType
}